packages/react-saasify/src/themes/waves/waves.module.css

```diff
@@ -271,142 +338,150 @@ header.container {
 }
 
 @media (min-width: 48em) {
-  header.container {
+  .theme-waves header.container {
     position: sticky;
   }
 }
 
-header.container .links {
+.theme-waves header.container .links {
   flex-grow: 1;
 }
 
-header.container .link {
-  color: var(--color-fg) !important;
+.theme-waves header.container .link {
+  color: var(--hero-color) !important;
   font-size: 18px;
   font-weight: 500;
 }
 
-header.container .link:hover {
+.theme-waves header.container .link:hover {
   color: var(--color-primary) !important;
   filter: var(--filter-tint);
 }
 
 @media (min-width: 48em) {
-  header.container .links .link {
+  .theme-waves header.container .links .link {
     margin-left: var(--space-xl);
     margin-right: var(--space-xl);
   }
 }
 
-header.container .links .link:first-of-type {
+.theme-waves header.container .links .link:first-of-type {
   margin-left: 0;
 }
 
-header.container .links .link:nth-of-type(2),
-header.container .links .link:nth-of-type(5) {
+.theme-waves header.container .links .link:nth-of-type(4) {
   display: none;
 }
 
-header.container .links .link:nth-of-type(3) {
+.theme-waves header.container .links .link:nth-of-type(2) {
```

Comment:
just a minor thought; in the future, we should support scss and/or lesscss for these themes so you can just wrap the top-level in the theme name instead of repeating it every time.

Reply:
wouldn't hurt - I personally dislike sass but it would certainly help scope here. Another option would be to treat as a module so themes don't cross pollute.